cordova-lib/src/plugman/install.js

```diff
@@ -404,6 +418,20 @@ function installDependencies(install, dependencies, options) {
     return dependencies.reduce(function(soFar, dep) {
         return soFar.then(
             function() {
+                // Split @Version from the plugin id if it exists.
+                var splitVersion = dep.id.split('@');
+                //Check if a mapping exists for the plugin id
+                //if it does, convert id to new name id 
+                var newId = pluginMapper[splitVersion[0]];
+                if(newId) {
+                    events.emit('log', 'Notice: ' + dep.id + ' has been automatically converted to ' + newId + ' and fetched from npm. This is due to our old plugins registry shutting down.');
```

Comment:
Shouldn't this be emitted with `warn` level to encourage users to switch to new plugin id instead of relying on plugin mapper?